setup.sh

```diff
@@ -193,14 +193,15 @@ function usage()
   say 'usage: ./setup.sh option'
   say ''
   say 'Where option may be one of the following:'
-  say '--help      | -h     Display this usage message'
-  say "--install   | -i     Install $app_name"
-  say "--uninstall | -u     Uninstall $app_name"
-  say '--skip-checks        Skip checks (use this when packaging)'
-  say '--verbose            Explain what is being done'
-  say '--force              Never prompt'
-  say "--completely-remove  Remove $app_name and all files under its responsibility"
-  say "--docs               Build $app_name's documentation as HTML pages into ./build"
+  say '--help                    | -h     Display this usage message'
+  say "--install                 | -i     Install $app_name"
+  say "--uninstall               | -u     Uninstall $app_name"
+  say "--setup-env-notification  | -s     Show current env in terminal"
+  say '--skip-checks             Skip checks (use this when packaging)'
+  say '--verbose                 Explain what is being done'
+  say '--force                   Never prompt'
+  say "--completely-remove       Remove $app_name and all files under its responsibility"
```

Comment:
When creating the new commit that changes this part, use the string concatenation style defined by kw.

Reply:
nice!!